__tests__/topic_test.js

```diff
@@ -0,0 +1,34 @@
+const request = require("supertest");
+const db = require("../db/connection.js");
+const testData = require("../db/data/test-data/index.js");
+const seed = require("../db/seeds/seed.js");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+const app = require("../app");
+
+  describe("GET/api/topics", () => {
+    test("status:200 - responds with an array of topics, with slug and description properties", () => {
+        return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then(({ body }) => {
+                const { topics } = body;
+                expect(topics).toBeInstanceOf(Array);
+                expect(topics).toHaveLength(3);
+                topics.forEach((topic) => {
+                    expect(topic).toMatchObject({
+                        slug: expect.any(String),
+                        description: expect.any(String),
+                    });
+                });
+            })
+        })
+        it("responds with a a 404,and invalid url ", () => {
```

Comment:
I'd test that you are getting a response aswell as the status.
Also where is it that this is being handeled I don't see an app.use set up to deal with a bad route.